src/Utils/Transport.php

```diff
@@ -38,9 +39,29 @@ public static function parseTransport(?string $transport): int
             return static::REST;
         }
         if ($transport === "grpc") {
-            throw new \Exception("gRPC-only PHP clients are not supported at this time");
+            return static::GRPC;
         }
 
         throw new \Exception("Transport $transport not supported");
     }
+
+    public static function isRestOnly(int $transport): bool
+    {
+        return Transport::compareTransports(Transport::REST, $transport);
+    }
+
+    public static function isGrpcOnly(int $transport): bool
+    {
+        return Transport::compareTransports(Transport::GRPC, $transport);
+    }
+
+    public static function isGRPCRest(int $transport): bool
+    {
+        return Transport::compareTransports(Transport::GRPC_REST, $transport);
+    }
+
+    private static function compareTransports(int $transportA, int $transportB): bool
+    {
+        return $transportA === $transportB;
+    }
```

Comment:
nit: do we really need this method? I think it's just as readable (and I would say more readable since there's fewer levels of redirection) to just do the comparison in the methods themselves.

Reply:
Hmmm is not necessary. I prefer to do it this way, for me sounds weird to have to repeat the same code and it has the added benefit of making it public in the future if we need it.

That being said, that opinion is just an opinion and I will not die on this hill for this. I can change it.

Reply:
I totally agree it's just a matter of opinion, and I would also prefer to die on a different hill.